modules/accounts/calendar.nix

```diff
@@ -144,6 +144,7 @@ in {
         (import ../programs/vdirsyncer-accounts.nix)
         (import ../programs/khal-accounts.nix)
         (import ../programs/khal-calendar-accounts.nix)
+        (import ../programs/qcal-accounts.nix)
```

Comment:
Should be possible to skip this line (and delete `qcal-accounts.nix`)

```suggestion
```

if you add

```nix
accounts.calendar.accounts = mkOption {
  type = with types;
    attrsOf (submodule {
      options.qcal.enable = mkEnableOption "qcal access";
    });
};
```

to the `options` attribute of `qcal.nix`.

Reply:
Thanks for the suggestion, done